### PR TITLE
docs: add information about provider tokens with social login

### PR DESCRIPTION
**Pull request by hf**
Explains provider tokens when used with social login.

### CODE REVIEW

**Review by J0 — APPROVED** [review]

**Review by kiwicopple — APPROVED** [review]

File: apps/docs/pages/guides/auth/social-login.mdx [hunk 1]
```diff
@@ -41,6 +41,12 @@ Supabase supports a suite of social providers. Follow these guides to configure
       </a>
     </Link>
 ))}
+  
+## Provider Tokens
```
Comment (kiwicopple) [inline apps/docs/pages/guides/auth/social-login.mdx:45]: @hf Should this be below the `</div>?

Comment (MildTomato) [inline apps/docs/pages/guides/auth/social-login.mdx:45]: OK to move this outside of that closing </div> below?

Comment (hf) [reply apps/docs/pages/guides/auth/social-login.mdx:45]: Oh my bad. Didn't see the </div>

Comment (hf) [reply apps/docs/pages/guides/auth/social-login.mdx:45]: Oh my bad. Didn't see the </div>

Comment (MildTomato) [reply apps/docs/pages/guides/auth/social-login.mdx:45]: ha, beat me to it

**Review by kiwicopple — CHANGES_REQUESTED** [review]
just want to check this doesn't mess up the formatting

File: apps/docs/pages/guides/auth/social-login.mdx (hunk 1 above)
Comment (hf) [reply apps/docs/pages/guides/auth/social-login.mdx:47]: LOL github does not allow me to write `</div>` :D :D :D

Comment (MildTomato) [reply apps/docs/pages/guides/auth/social-login.mdx:47]: @hf oh yes, happens to me all the time!

**Review by kiwicopple — APPROVED** [review]

**Pull request merged**